server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -307,7 +313,7 @@ private Modal renderDownloadApplicationsModal(
         .build();
   }
 
-  private DivTag renderApplicationListItem(Application application) {
+  private DivTag renderApplicationListItem(Application application, boolean supportsStatuses) {
```

Comment:
'supports' reads a little odd, maybe name it 'displayStatuses' or have a javadoc that clarifies the implication of the param.

Reply:
Done!